### ISSUE TITLE
Transcription works on cli but it does not in code

### ISSUE THREAD

**Issue by jpsanr** [jdepoix/youtube-transcript-api#15]
Hi i'm facing this exception when I try to transcript a video: 

> Could not get the transcript for the video https://www.youtube.com/watch?v=1zCs7zYZK4g! Most likely subtitles have been disabled by the uploader or the video is no longer available.

But if I try to use by cli version it works perfectly.

Any ideas of what I can do ?

My code:

>from youtube_transcript_api import YouTubeTranscriptApi
x = YouTubeTranscriptApi.get_transcript(["1zCs7zYZK4g"], languages=["en"])
print(x)

**Comment by jdepoix** [comment]
If you use `get_transcript` you should only provide a single ID instead of a list. If you want to pass in a list you should use `get_transcripts`.

This should work:
```python
from youtube_transcript_api import YouTubeTranscriptApi
x = YouTubeTranscriptApi.get_transcript("1zCs7zYZK4g", languages=["en"])
print(x)
```

**Comment by jpsanr** [comment]
Hi I still having problems: 
If I run the code as the following it works fine:

> x = (YouTubeTranscriptApi.get_transcript("pQIko8-lmLM", languages=["en"]))
print(x)

But if I try to run into a method it raise an exception:

> def get_video_transcription(video_id):
    print(video_id)
    transcription_text = YouTubeTranscriptApi.get_transcript(video_id, languages=["pt"])
    return transcription_text



> lista = ["pQIko8-lmLM", "qxrtk_UoVcc"]
for l in lista:
    print("************** {} **********".format(l))
    x =(get_video_transcription(l))
    print(x)
    
Any ideas ?
Thank you

**Comment by jdepoix** [comment]
@jpsanr it looks like the language "pt" is not supported for this video, as it works with `languages=["en",]`. Unfortunately there's nothing you or I could do about this. 

However, I will add this possibility to the `CouldNotRetrieveTranscript` error message.

**Comment by jdepoix** [comment]
Added the improved error message in #16 and released with v0.1.5
